glass/fields.py

```diff
@@ -44,11 +42,9 @@ def inv_triangle_number(triangle_number: int) -> int:
 
 def iternorm(
     k: int,
-    cov: collections.abc.Iterable[npt.NDArray[np.float64]],
+    cov: Iterable[NDArray[np.float64]],
     size: int | tuple[int, ...] = (),
-) -> collections.abc.Generator[
-    tuple[int | None, npt.NDArray[np.float64], npt.NDArray[np.float64]]
-]:
+) -> Generator[tuple[int | None, NDArray[np.float64], NDArray[np.float64]]]:
```

Comment:
Not really a comment on the PR as such, but the unary `Generator` here and below could be changed to `Iterator` which is the same.